rank/pages/api/search.ts

```diff
@@ -38,12 +35,13 @@ export default async (
           ...template.source,
           track_total_hits: true,
           highlight: {
-            pre_tags: ['<em class="bg-yellow-200">'],
-            post_tags: ['</em>'],
+            pre_tags: ['<span class="bg-yellow-200">'],
```

Comment:
Why?

Reply:
Purely because it's a bit easier on my eyes... I found myself squinting and rereading the highlights multiple times, when they were also italicised, but the highlight by itself seemed fine. 

Happy to revert if you want though?